delivery/delete-cloud-installation/README.md

```diff
@@ -0,0 +1,30 @@
+# delete-cloud-installation
+
+This action is used to delete a Mattermost cloud installation from Github Actions.
+
+### Usage
+
+```yaml
+jobs:
+  create-cloud-installation:
+    runs-on: <private-runners>
+    steps:
+      - uses: mattermost/actions/delivery/install-mattermost-cloud@main
+        with:
+          version: 0.83.0
+          
+      - uses: mattermost/actions/delivery/delete-cloud-installation@main
+        with:
+          server: https://delivery-provisioner.test.mattermost.com
+          installation-id: <my-installation-id> 
+          headers: |-
+            x-api-key=MY_TOKEN
+```
+
+### Inputs
+
+| Input           | Description                                                                                        | Default | Required |
+| --------------- | -------------------------------------------------------------------------------------------------- | ------- | -------- |
+| server          | The provisioning server whose API will be queried.                                                 |         | true     |
+| headers         | The headers to add in every API call towards the provisioning server. Accepts new line list format |         | false    |
```

Comment:
Same comment here if server and header need to be exposed to consumer or just declare within the action itself.

Reply:
Yep it needs to be exposed . This is a reusable action and might be used in different places